vision/google/cloud/vision/_http.py

```diff
@@ -52,6 +52,8 @@ def annotate(self, image, features):
         images = api_response.get('responses')
         if len(images) == 1:
             return Annotations.from_api_repr(images[0])
+        elif len(images) > 1:
+            raise ValueError('Multiple image processing is not yet supported.')
```

Comment:
1. Based on the message it seems like `NotImplementedError` could be just as appropriate
1. And `if len(images) == 0`, that's OK?